example/src/ConversationView.tsx

```diff
@@ -8,25 +9,44 @@ import {
   View,
   Button,
 } from "react-native";
-import { Conversation, DecodedMessage } from "xmtp-react-native-sdk";
+import { Conversation } from "xmtp-react-native-sdk";
 
 import { RootStackParamList } from "./HomeView";
+import { TextCodec } from "./test_utils";
+import {
+  CodecRegistry,
+  ContentCodecInterface,
+} from "../../src/lib/CodecRegistry";
 
 type Props = NativeStackScreenProps<RootStackParamList, "Conversation View">;
 
+type DecodedMessage = {
+  content: string;
```

Comment:
I think this should be `any` now that content can be whatever there's a content codec for?
```suggestion
  content: any;
```